config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.yml

```diff
@@ -19,6 +19,7 @@ Limber-Htp - scRNA Core cDNA Prep GEM-X 5p:
 Limber-Htp - scRNA Core cDNA Prep GEM-X 5p GEMs Input:
   submission_class_name: "LinearSubmission"
   related_records:
-    request_type_keys: ["limber_scrna_core_cdna_prep_gem_x_5p_gems_input"]
+    request_type_keys:
+      ["limber_scrna_core_cdna_prep_gem_x_5p_gems_input", "limber_scrna_core_cdna_prep_gem_x_5p_gems_input_cite"]
```

Comment:
Check also the closing of this CITE request type.
See 
```
LRC GEM-X 5p cDNA PCR XP:
  :asset_type: plate
  :stock_plate: false
  :input_plate: false
  # close off the submission when the plate is created
  :state_changer_class: StateChangers::AutomaticPlateStateChanger
  :work_completion_request_type:
    ['limber_scrna_core_cdna_prep_gem_x_5p_gems_input', 'limber_scrna_core_cdna_prep_gem_x_5p']
```
 Do we need to add the cite one to that array.